app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+from pathlib import Path
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        database_path = Path("..") / "cinema.sqlite"
```

Comment:
in this task you can simply use `sqlite3.connect("cinema.sqlite")` considering your database is in app/ directory. Also you absolute path with preferable not realtive